src/test/java/LoginTest.java

```diff
@@ -1,34 +1,190 @@
 import org.testng.annotations.Test;
 import pages.LoginPage;
 import util.Constants;
 
 import java.time.Duration;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static util.Constants.DOWNLOAD2_URL;
+import static util.Constants.DOWNLOAD_URL;
 
 
 public class LoginTest {
     WebDriver driver;
 
+    public static final By clickMeBtn = By.xpath("//button[@class='btn btn-default btn-lg' and text() = 'Click me!']");
+    public static final By clickAlertBtn = By.xpath("//button[contains(text(), 'Prompt Box')]");
+    public static final By clickMeRandomBtn = By.xpath("//button[@class='btn btn-default']");
+    public static final By percentage = By.xpath("//div[@class='percenttext']");
+    public static final By clickBtnPercent = By.id("cricle-btn");
+    public static final By newUser = By.xpath("//div[@id='loading']/img");
+    public static final By resultMsg = By.id("confirm-demo");
+    public static final By resultMsgAlert = By.id("prompt-demo");
+    public static final By states = By.name("States");
+
+
     @BeforeMethod(alwaysRun = true)
     public void preparingData() {
         WebDriverManager.getInstance(ChromeDriver.class).setup();
         driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(30));
     }
 
-    @Test
-    public void loginTest() {
+    @DataProvider(name = "authentication")
+    public Object[][] dpMethod() {
+        return new Object[][]{
+                {Constants.LOGIN_USER, Constants.PASSWORD_USER},
+                {Constants.LOGIN_USER2, Constants.PASSWORD_USER2}};
+    }
+
+    @Test(dataProvider = "authentication")
+    public void loginTest(String username, String password) throws InterruptedException {
         driver.get(Constants.YANDEX_URL);
         LoginPage loginPage = new LoginPage(driver);
-        loginPage.login(Constants.LOGIN_USER, Constants.PASSWORD_USER);
+        loginPage.login(username, password);
         Assert.assertTrue(loginPage.isComposeButtonDisplayed());
     }
 
+    @Test
+    public void multiSelectTest() {
+        driver.get(Constants.MULTI_SELECT_URL);
+        WebElement selectElement = driver.findElement(states);
+        Select select = new Select(selectElement);
+
+        List<String> expectedOptions = getRandomElements(3, select.getOptions().stream().map(WebElement::getText).collect(Collectors.toList()));
+        expectedOptions.forEach(select::selectByVisibleText);
+        List<String> selectedOptions = (select.getAllSelectedOptions().stream().map(WebElement::getText).collect(Collectors.toList()));
+
+        Collections.sort(selectedOptions);
+        Collections.sort(expectedOptions);
+
+        Assert.assertEquals(selectedOptions, expectedOptions);
+    }
+
+    public List<String> getRandomElements(int amount, List<String> list) {
+        ArrayList<String> returnList = new ArrayList<String>();
+        Random rand = new Random();
+
+        for (int i = 0; i < amount; i++) {
```

Comment:
too complicated logic. Try ints() method of Random class. Your method may look like:
```
public List<String> getRandomElements(int amount, List<String> list) {
        return new Random().ints(amount, 0, list.size()).boxed()
                .map(i -> list.get(i)).collect(Collectors.toList());
    }
```